lib/collective.lib.js

```diff
@@ -49,6 +49,11 @@ export const getCollectiveMainTag = (hostCollectiveId = null, tags = [], type, s
   return CollectiveCategory.COLLECTIVE;
 };
 
+export const expenseSubmissionAllowed = (collective, user) => {
+  const member = user?.memberOf.filter(member => member.collective.slug === collective.slug);
```

Comment:
:ice_cream: Small perf improvement: we don't need to filter the entire list of members if `collective.settings.disableExpenseSubmission` is false, which is the majority of the cases.

I would start the function with a `if (!collective?.settings.disableExpenseSubmission) return true;`

Reply:
Done. 👍🏽